### DISCUSSION TITLE
Any updates on the slider?

### DISCUSSION THREAD

**Discussion by samarthpuri** [twilio-labs/paste#1627] [Ideas] [answered]
Saw this [post](https://github.com/twilio-labs/paste/discussions/976) from November and was just curious if there were any updates on the slider? Currently utilizing SendGrid's with Paste's colors.

**Comment by nkrantz** [comment, marked as answer]
Hi @samarthpuri, no updates on the Slider component since Simon's comment on the post you referenced. 

A bit of insight into how we plan our releases. We do regular prioritization planning with a variety of teams, in which we get a sense for which components are in high demand across the board. We then run those scores against the ratings we've assigned each component regarding level of difficulty to design/build. We also weigh the impact that any given component would have on each team at Twilio. From there, we choose the highest priority components for each upcoming release.

I hope this provides some transparency! Let me know if you have any questions.

**Comment by ShaunHalliday** [comment]
Hi All, just to contribute we have also had a request for a slider component from a customer/client as part of a UI, I can see on the roadmap this is in the backlog.

**Reply by SiTaggart** [reply]
This is still the case for now, we'll let you know if anything changes in regard to prioritizing it. 

We did toss the idea around of expediting it recently with a really simple version, but some company things happened and it got prioritized again. We might look to pull that back up again when things settle a little.

**Reply by ShaunHalliday** [reply]
Thanks Si, just thought it good to throw some more weight behind it to hopefully push it up the roadmap list 😄
